src/app/(clerk)/my-portfolio/manual-edit/page.js

```diff
@@ -0,0 +1,89 @@
+"use client";
+
+import { useState } from "react";
+
+const ManualEditPage = () => {
+  const [formData, setFormData] = useState({
+    FirstName: "",
+    LastName: "",
+    NavName: "",
+    NavSubtitle: "",
+    image: "",
+    AboutImageLeft: "",
+    AboutImageCenter: "",
+    AboutImageRight: "",
+    linkedinUrl: "",
+    resumeLink: "",
+    githubId: "",
+    email: "",
+    WhatsAppLink: "",
+    HeroGradient: "",
+    HeroTitle: "",
+    HeroSubtitle: "",
+    AboutTitle: "",
+    AboutTitleGradient: "",
+    AboutMockCode: "",
+    AboutBadge: "",
+    AboutDescription: "",
+    AboutMainDescription: "",
+    AboutOtherDescription: "",
+    ProjectTitle: "",
+    ProjectTitleGradient: "",
+    ProjectDescription: "",
+    ProjectBadge: "",
+    SkillsBadge: "",
+    FooterDescription: "",
+    FooterBadge: "",
+    FootTitle: "",
+    FooterSliderTitle: "",
+  });
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setFormData({ ...formData, [name]: value });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log("Generated JSON:", JSON.stringify(formData, null, 2));
+    alert("JSON generated! Check the console for details.");
+  };
```

Comment:
Handle update portfolio code in firestore!
[Function to do the same](https://github.com/hi-Kartik2004/CraftFolio/blob/5d44d514185cb7571d6a612f3956b6369ec373ab/src/app/components/EditPortfolio.jsx#L112C1-L148C4)